linux/CodeCoverageReports.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<head>
+  <title>Keyman for Linux Code Coverage</title>
+  <style type="text/css">
+    body, html {width: 100%; height: 100%; margin-left: 0; padding-left: 0;}
+    .row-container {display: flex; width: 100%; height: 100%; flex-direction: column; overflow: hidden;}
+    .second-row { flex-grow: 1; border: none; margin: 0; padding: 0; border: none; }
+    .tablink { background-color: #555; color: white; float: left; border: none; border-right: 1px solid black; outline: none; cursor: pointer; padding: 14px 16px; font-size: 17px; width: 33.3%; }
+    .tablink:hover { background-color: #777; }
+  </style>
+</head>
+  <body>
+  <div class="row-container">
+    <div class="first-row">
+      <h1>Keyman for Linux Code Coverage Reports</h1>
+      <button class="tablink" onclick="openPage(this, 'build/x86_64/debug/meson-logs/coveragereport/index.html')" id="defaultOpen">ibus-keyman</button>
+      <button class="tablink" onclick="openPage(this, 'keyman-config/build/coveragereport/index.html')">keyman-config</button>
+      <button class="tablink" onclick="openPage(this, 'keyman-system-service/build/x86_64/debug/meson-logs/coveragereport/index.html')">keyman-system-service</button>
+    </div>
+    <iframe id="iframe_a" class="second-row"></iframe>
+  </div>
+
+  <script>
+    function openPage(elmnt, filename) {
+      document.getElementById('iframe_a').src = filename;
+      tablinks = document.getElementsByClassName("tablink");
+      for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].style.backgroundColor = "";
+      }
+      elmnt.style.backgroundColor = "#777";
+    }
```

Comment:
I wonder if it would be better for the reports to just be three plain links in this file, `<li><a href='.../index.html'>ibus-keyman</a></li>`

That's less to maintain and easier to share a specific report